.gitmodules

```diff
@@ -4,6 +4,9 @@
 [submodule "rabit"]
 	path = rabit
 	url = https://github.com/dmlc/rabit
+[submodule "nccl"]
+	path = nccl
+	url = https://github.com/h2oai/nccl
```

Comment:
Great, I put in a PR to that new repo.  Once that's accepted, I'll be able to change the h2oai -> dmlc for the submodule.  Thanks!